Assets/scripts/MenuManager.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MenuManager : MonoBehaviour
+{
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
+
+    public void StartGame()
+    {
+        SceneManager.LoadScene(2);
```

Comment:
Use string, not id, because if we change the order or we just remove the sample scene, it will broken.
Plus no one knows, which one is the 2.